### DISCUSSION TITLE
北京市郊铁路相关信息补充

### DISCUSSION THREAD

**Discussion by ma-er233** [Ivysauro/CNRT#63] [Ideas]
虽然是国铁运营，但是实际功能上和地铁差不多，是去往北京郊区的较优选择，感觉可以补充一下
![IMG_1651](https://github.com/Ivysauro/CNRT/assets/79030457/7d9a6526-c532-4828-aa76-fd81bd077905)
如图，这几条线路里 S2 线不需要购票，可直接刷市政交通一卡通（实体卡或NFC都可以）、交通联合卡、亿通行（无需验证）、北京公交APP、北京一卡通APP等，忘了支不支持微信支付宝，总之乘坐方式基本和地铁一样。此外这条线也比较有历史价值。
另外几条线可以在 12306 或线下购票（但是会售空，后面几种即使 12306 上售空了也可以坐），或用亿通行（需要人码合一认证）、市郊铁路一卡通等
市郊铁路发车情况随周末、节假日、游客数量、天气等情况变动很大，具体时刻表可以看市郊铁路的公众号

另外，请问索引→特殊运营方式里的“北京地铁副中心2号线”是哪一条？是和上述市郊铁路有关的线路还是地铁？查不大到这条线的信息

**Comment by Ivysauro** [comment]
市郊铁路的确可以补充到支付方式之中，但收录哪些有待研究
例如，收录北京S2，则非实名制的上海金铁、连云港市郊铁路是类似的可收录。收录北京其他实名制市郊副中心、怀密等（可扫码乘车、可12306购票），那是否可以收录广东城际运营的广清城际等城际线路（可扫码乘车、可12306购票）？如果可以收录广东城际运营的城际线路，那是否可以收录国铁运营的城际线路呢（可使用铁路e卡通扫码、可12306购票）？😂

**Comment by Ivysauro** [comment]
```
另外，请问索引→特殊运营方式里的“北京地铁副中心2号线”是哪一条？
```
有些内容未更新，“副中心2号线”现在名称应该是“M102线”

**Reply by ma-er233** [reply]
原来如此，感谢

上面说的确实是个问题，我主要是因为住在 S2 线附近，对这几条线谜之有热情 🙈

**Reply by Ivysauro** [reply]
> 原来如此，感谢
> 
> 
> 
> 上面说的确实是个问题，我主要是因为住在 S2 线附近，对这几条线谜之有热情 🙈

后续可能会加入这些非实名制的市域线路

**Comment by chenw23** [comment]
已复核并纳入 PR #153。

1. 讨论中提及的“副中心2号线”是旧称；北京市政府当前使用 `M102线`，并将其定位为副中心“一环”，故已更正名称并移出错误的“其他支线”分类。
2. 市郊铁路 S2、城市副中心、怀柔—密云、通密线已可用实体一卡通、北京一卡通App（含NFC）、支付宝/亿通行乘车码与地铁一票通行、合并累计优惠；S2还支持北京互通卡和市郊铁路一卡通。

PR：https://github.com/Ivysauro/CNRT/pull/153
依据：https://www.beijing.gov.cn/ywdt/gzdt/202105/t20210513_2388074.html
https://www.beijing.gov.cn/fuwu/bmfw/sy/jrts/tzxx/202312/t20231228_3517440.html
https://www.beijing.gov.cn/renwen/zt/kwctdlc/yw/gpfs/202404/t20240410_3614569.html

本仓库的主数据是城市轨道交通信息库；市郊铁路仅补入与地铁直接关联的一票通行信息，未扩展为完整国铁/市郊铁路数据库。
